### PR TITLE
Github issues Actions & Reactions

### PR DESCRIPTION
**Pull request by EdenComp**
Also fixes ARE-324
Also fixes ARE-325
Also fixes ARE-326

### CODE REVIEW

**Review by RezaRahemtola — APPROVED** [review]

File: backend/back/src/webhooks/services/gihub-webhook.service.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { GrpcService } from "../../grpc/grpc.service";
 import { createHmac, timingSafeEqual } from "crypto";
 import { ConfigService } from "@nestjs/config";
 
-type GithubActions = "opened";
+type GithubActions = "opened" | "reopened" | "closed";
```
Comment (Croos3r) [inline backend/back/src/webhooks/services/gihub-webhook.service.ts:6]: Is "GithubActions" a good name ? Could GithubIssueState be a better name ? It's a type (not an array), action is not representing the underlying value and its the state of something, not of GitHub globally

File: backend/back/src/webhooks/services/gihub-webhook.service.ts [hunk 2]
```diff
@@ -42,20 +53,22 @@ export class GithubWebhookService {
...
+						url: body.issue.html_url,
+						title: body.issue.title,
+						body: body.issue.body,
+						createdAt: body.issue.created_at,
+						author: body.sender.login,
+					},
+				});
 			}
 		}
+		console.log(body);
```
Comment (Croos3r) [inline backend/back/src/webhooks/services/gihub-webhook.service.ts:71]: Is this line necessarry ?

File: backend/back/src/workflows/seed/1698331915608-CreateGithubIssuesAreas.ts
Comment (Croos3r) [inline backend/back/src/workflows/seed/1698331915608-CreateGithubIssuesAreas.ts:1]: You could have used constant variables and looped over them for this migration but it's okay

**Review by Croos3r — CHANGES_REQUESTED** [review]

File: backend/back/src/webhooks/services/gihub-webhook.service.ts (hunk 2 above)
Comment (EdenComp) [reply backend/back/src/webhooks/services/gihub-webhook.service.ts]: Done

File: backend/back/src/webhooks/services/gihub-webhook.service.ts (hunk 1 above)
Comment (EdenComp) [reply backend/back/src/webhooks/services/gihub-webhook.service.ts]: Done

Comment (EdenComp) [reply backend/back/src/webhooks/services/gihub-webhook.service.ts]: (named it GithubBodyActions as the key is name "action" and is used in all Github payloads: issues, pr, commits...)

**Pull request merged**